data/json/items/tool/tailoring.json

```
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "awl_bone",
+    "type": "GENERIC",
+    "category": "tools",
+    "name": "bone sewing awl",
+    "description": "This is a crude sharpened bone awl; those were used for leatherworking before metal was discovered.  It can also serve as an improvised stabbing weapon.",
+    "weight": 200,
+    "volume": "250 ml",
+    "price": 300,
+    "to_hit": -1,
+    "bashing": 2,
+    "cutting": 5,
+    "material": "bone",
+    "symbol": ";",
+    "color": "yellow",
+    "qualities": [ [ "LEATHER_AWL", 1 ] ],
+    "flags": [ "SPEAR", "BELT_CLIP" ]
```

Per what @kasanryukin said. 
```suggestion
    "flags": [ "SPEAR", "BELT_CLIP", "FRAGILE_MELEE ]
```